src/main/java/cz/cvut/kbss/termit/rest/VocabularyController.java

```diff
@@ -129,6 +129,17 @@ private URI resolveVocabularyUri(String fragment, String namespace) {
         return vocabularyService.getChangesOfContent(vocabulary);
     }
 
+    /**
+     * Gets the number of terms in the vocabulary with the specified identification
+     */
+    @PreAuthorize("permitAll()")
+    @GetMapping(value = "/{fragment}/terms/count")
```

Comment:
Is this going to collide with a term with fragment 'count'? 

We have some other occurrences of this at other places of the code, but we should not proliferate this issue.

Why not to use e.g. X-Total-Count header ?

Reply:
I like the X-Total-Count header solution, there is just one problem: the tree select, which loads terms from the backend, uses the `/roots` endpoint. In that case, returning the total number of terms would IMHO be misleading and returning just the number of roots would not be sufficient for the functionality.

Reply:
How is this ticket connected to the tree select component?  Isn't it just about some badge with the count in the tab outside of the tree select?